test/libsolidity/semanticTests/array/push/push_no_args_bytes.sol

```diff
@@ -23,7 +23,7 @@ contract C {
 // ----
 // l() -> 0
 // g(uint256): 70 ->
-// gas irOptimized: 428769
+// gas irOptimized: 434229
```

Comment:
This is due to added check for array size is less then max array size

Reply:
We need to find a way to pull this out of the loop...